data/data/ovirt/.terraform/plugin_path

```diff
@@ -0,0 +1,3 @@
+[
+  "/home/rgolan/go/bin/"
```

Comment:
This doesn't look right . . .

Reply:
Thanks for noticing this as well - I removed the .terraform folder completely.

Reply:
I removed it till I figured out a better local unit test